### PR TITLE
Jenkins: update to run on Jenkins 2.0 with Blue Ocean pipelines

### PR DESCRIPTION
**Pull request by jmoody**
### Motivation

Completes:

* [Spike] use Jenkins 2.0 pipelines to automate testing and distribution of the calabash.framework [VSTS](https://msmobilecenter.visualstudio.com/Mobile-Center/_workitems/edit/39015)

* We need to migrate to Jenkins 2.0.
* Our existing tests push to Xamarin Test Cloud (RIP) which has been replaced by AppCenter
* We are looking for ways to automate the distribution the calabash.framework.

### CODE REVIEW

File: bin/ci/jenkins/appcenter.sh [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+appcenter test run calabash \
+  --app "App-Center-Test-Cloud/LPTestTarget-iOS" \
+  --devices "App-Center-Test-Cloud/latest-releases-ios" \
+  --app-path "${WORKSPACE}/LPTestTarget.ipa" \
+  --test-series "${SERIES}" \
+  --project-dir "${WORKSPACE}" \
+  --config-path "${WORKSPACE}/cucumber.yml" \
+  --token "${AC_TOKEN}" \
+  --dsym-dir "${WORKSPACE}/LPTestTarget.app.dSYM" \
```
Comment (polyakrecords) [inline bin/ci/jenkins/appcenter.sh]: Seems like this argument isn't working =\

File: bin/ci/jenkins/appcenter.sh [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  error ""
+  error "Then try again."
+  exit 1
+fi
+
+CAL_CODESIGN="${HOME}/.calabash/calabash-codesign"
+if [ -e "${CAL_CODESIGN}" ]; then
+  AC_TOKEN=$("${CAL_CODESIGN}/apple/find-appcenter-credential.sh" api-token)
+else
+  echo "here"
```
Comment (polyakrecords) [inline bin/ci/jenkins/appcenter.sh]: Do we need this `echo`? =)

File: bin/ci/jenkins/appcenter.sh (hunk 1 above)
Comment (jmoody) [reply bin/ci/jenkins/appcenter.sh]: Let's leave this for now.

File: bin/ci/jenkins/appcenter.sh (hunk 2 above)
Comment (jmoody) [reply bin/ci/jenkins/appcenter.sh]: No. 🥇

**Pull request merged**